src/plugins/notebook/utils/notebook-image.js

```diff
@@ -43,30 +43,16 @@ export function getThumbnailURLFromimageUrl(imageUrl, size = DEFAULT_SIZE) {
 
     image.onload = function () {
       canvas.getContext('2d').drawImage(image, 0, 0, size.width, size.height);
-
       resolve(canvas.toDataURL('image/png'));
     };
 
     image.src = imageUrl;
   });
 }
 
-export function saveNotebookImageDomainObject(openmct, object) {
-  return new Promise((resolve, reject) => {
-    openmct.objects
-      .save(object)
-      .then((result) => {
-        if (result) {
-          resolve(object);
-        } else {
-          reject();
-        }
-      })
-      .catch((e) => {
-        console.error(e);
-        reject();
-      });
-  });
+export async function saveNotebookImageDomainObject(openmct, object) {
+  console.debug('🍎 Saving notebook image domain object', object);
+  await openmct.objects.save(object);
```

Comment:
could this cause issues if one user is editing an entry (has an open transaction) and the other is adding an image to an entry? we should test that

Reply:
I think this working ok:

https://github.com/nasa/openmct/assets/9853862/a91932ba-8c92-4ed2-b0a8-03d2918d1cf6